7/zts/vendor/karibbu/Dockerfile

```diff
@@ -8,6 +8,7 @@ request_order=GP\n\
 expose_php=Off\n\
 enable_dl=Off\n\
 short_open_tag=Off\n\
+zend.assertions=-1\n\
```

Comment:
Ok better setting for production... but we should activate assertions in dev/test